snorkel/labeling/lf/core.py

```diff
@@ -1,6 +1,6 @@
 from typing import Any, Callable, List, Mapping, Optional
 
-from snorkel.labeling.preprocess import BasePreprocessor
+from snorkel.preprocess import BasePreprocessor
```

Comment:
should we just change move all of this to `snorkel.map` instead of `snorkel.preprocess` and remove the indirection?

Reply:
`map` is pretty vague—we could merge `map` into `snorkel.preprocess`